Add Consent Amendment History feature (#35)
wso2/docs-ob-toolkit-cds: en/docs/install-and-setup/setting-up-databases.md

    
    - `openbank_ob_reporting_statsdb`

> 3. If you are using the [Consent Amendment History](../learn/consent-amendment-history.md) feature, it is necessary to create a separate database:

Reviewer: ```suggestion
3. If you are using the [Consent Amendment History](../learn/consent-amendment-history.md) feature, create the following database and execute the given scripts:
```

    - `ob_consent_history`

    - Sample table creation script is given below:

      ```tab="MySQL"
      CREATE TABLE IF NOT EXISTS OB_CONSENT_HISTORY (
         TABLE_ID VARCHAR(10) NOT NULL,
         RECORD_ID VARCHAR(255) NOT NULL,
         HISTORY_ID VARCHAR(255) NOT NULL,
         CHANGED_VALUES JSON NOT NULL,
         REASON VARCHAR(255) NOT NULL,
         EFFECTIVE_TIMESTAMP BIGINT NOT NULL,
         PRIMARY KEY (TABLE_ID,RECORD_ID,HISTORY_ID)
      )
      ENGINE INNODB;
      ```
      

>       ```tab="MsSQL"

Reviewer: ```suggestion
      ```tab="MS SQL"
```

      CREATE TABLE OB_CONSENT_HISTORY (
        TABLE_ID VARCHAR(10) NOT NULL,
        RECORD_ID VARCHAR(255) NOT NULL,
        HISTORY_ID VARCHAR(255) NOT NULL,
        CHANGED_VALUES NVARCHAR(max) NOT NULL,
        REASON VARCHAR(255) NOT NULL,
        EFFECTIVE_TIMESTAMP BIGINT NOT NULL,
        PRIMARY KEY (TABLE_ID,RECORD_ID,HISTORY_ID)
      );
      ```

      ```tab="Oracle"
      CREATE TABLE OB_CONSENT_HISTORY (
        TABLE_ID VARCHAR(10) NOT NULL,
        RECORD_ID VARCHAR(255) NOT NULL,
        HISTORY_ID VARCHAR(255) NOT NULL,
        CHANGED_VALUES CLOB NOT NULL,
        REASON VARCHAR(255) NOT NULL,
        EFFECTIVE_TIMESTAMP NUMBER NOT NULL,
        PRIMARY KEY (TABLE_ID,RECORD_ID,HISTORY_ID)
      );
      ```

>       ```tab="PostgresQL"

Reviewer: ```suggestion
      ```tab="PostgreSQL"
```
